cmd/edit/cluster/clusterregistryconfig.go

```diff
@@ -0,0 +1,48 @@
+/**
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"fmt"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+
+	"github.com/openshift/rosa/pkg/input"
+)
+
+func BuildAdditionalTrustedCAFromInputFile(specPath string) (map[string]string, error) {
+	specJson, err := input.UnmarshalInputFile(specPath)
+	if err != nil {
+		return nil, fmt.Errorf("Expected a valid additional trusted certificate spec file: %v", err)
+	}
+	form := make(map[string]string)
+	var ok bool
+	for k, v := range specJson {
+		form[k], ok = v.(string)
+		if !ok {
+			return nil, fmt.Errorf("expect value to be string, got %T", v)
```

Comment:
Perhaps we could add a bit more detail here. Since we already have the key, it would be more convenient for the user if we included that as well.

I recommend printing the received value instead of just the type (or both, if possible).

All the other errors start with an uppercase letter. Is this because they are printed directly to the user?
If that's the case, this one should also start with an uppercase letter.

Otherwise, Go's best practice is to always return errors in lowercase, allowing them to be easily concatenated to log messages like:
```go
fmt.Println("An error has occurred: ", err)
```

Reply:
Thanks @ziccardi , will let me add more info to the error log & look into upper case